Sources/PackageLoading/PackageExtensions.swift

```diff
@@ -145,6 +145,15 @@ extension Package {
             }
         }
 
+        /// Check for targets that are not mapped to any modules
```

Comment:
nit: full stop at end of the comment and this should probably be a `//` (not entirely sure though...)